infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -1362,4 +1363,14 @@ public FileVisitResult visitFileFailed(Path file, IOException exc) {
             throw new IllegalStateException(ioe);
         }
     }
+
+    public long getSegmentSize(long startAddress) {
+        long size = 0;
+        try {
```

Comment:
Just let the exception bubble up, otherwise it can be confusing to debug `IOException` ' s.